fix(third_party): mark body as verbatim

dandhlee opened the pull request:
[cloud-asset](https://github.com/googleapis/python-asset) has some [documentation](https://github.com/googleapis/python-asset/blob/master/google/cloud/asset_v1/types/asset_service.py#L716-L717) which does not render well and needs to be marked as verbatim, for example:
```
    -  ``NOT state:ACTIVE`` to find {{gcp_name}} resources whose
       state doesn&#39;t contain &quot;ACTIVE&quot; as a word.
```
trying to stage `{{gcp_name}}` throws an error. This documentation is auto-generated so I think we should simply mark this kind of content as verbatim.

Also updating the section to only show when necessary.

> It's a good idea to open an issue first for discussion.

- [x] Tests pass
- [ ] Appropriate changes to README are included in PR

----

As discussed, changing entire body field as verbatim and removing snippets of verbatim tags inside.

dandhlee committed:
fix(third_party): mark summary as verbatim

dandhlee committed:
test: update goldens

dandhlee committed:
fix: only show content when necessary

tbpg commented on third_party/docfx/templates/devsite/partials/uref/namespace.tmpl.partial:
```diff
@@ -21,7 +21,11 @@
     {{#children}}
       <h2><xref uid="{{uid}}" altProperty="fullName" displayProperty="name"/></h2>
+      {{#summary}}
+      {% verbatim %}
```
We're playing Whack-a-Mole with `verbatim`. What if we mark everything inside the `body` tags as `verbatim` and remove the others?

dandhlee replied:
Looks like marking stuff verbatim has little to no side effect besides whitespaces we might be adding (if placed incorrectly) and more content being rendered (which should probably show?), so I think it sounds like a better idea.

dandhlee committed:
fix(third_party): consolidate verbatim tag to body

dandhlee committed:
test: update goldens

dandhlee commented:
Removed verbatim tags and consolidated under body ([see this commit](https://github.com/googleapis/doc-templates/pull/229/commits/cdceaa347fa07c2aad815cc0c922d11365dd9197)), updated test data.

Tried staging some files and all seems to be good. Please take a look again!

tbpg reviewed (changes requested):
(Talked offline, closing the loop here.) Need to run a little extra testing on this to make sure it does the right thing.

tbpg reviewed (approved):
Looks good.
